src/main/java/com/ohmyclass/api/components/comment/repository/ICommentRepository.java

```diff
@@ -1,5 +1,17 @@
 package com.ohmyclass.api.components.comment.repository;
 
-public interface ICommentRepository {
+import java.sql.Date;
+import java.util.Optional;
 
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import com.ohmyclass.api.components.comment.entity.Comment;
+
+@Repository
+public interface ICommentRepository extends CrudRepository<Comment, Long> {
+
+    Optional<Comment> findCommentById(Long id);
+
+    Optional<Comment> findCommentOrderByDate(Date date);
```

Comment:
Order what? A single comment? What are you "querying" for?
And why do you need a date? It's order by date; Not "findByDate".

Change to something like "List<Comment> findAllCommentsOrderByDate();"
But this isn't a good approach either. User "findAllCommentsByClassIdOrderByDate(Long classId)" as an inspiration.